### ISSUE TITLE
Deactivate stale assessment categories

### ISSUE THREAD

**Issue by Quinn-Donnelly** [revaturelabs/1808-cal-v2-meta#69]
As a VP, I can deactivate stale assessment categories

**Comment by akilisosa** [comment]
how would you define a stale assessment category?

**Comment by Quinn-Donnelly** [comment]
@akilisosa 
Internally we would know basically QC would review a curriculum change it then go look to remove any out of date categories that don't align with the new curriculum. You may assume the user has knowledge of what is stale.

**Comment by akilisosa** [comment]
Task 1: understand database design for assessment categories.  1hr.
Task 2: get the back end services available. 1hr.
Task 3: Explore existing Angular Application.  2hr.
Task 4: coordinate with other team 8hr.
Task 5: collaborate with teammates 8hrs.
Task 6: testing with Jasmine 4hrs.
Task 7:implementation 4 hrs. 
task 8: correct documentation, pushing to dev 6hrs.

**Comment by akilisosa** [comment]
task 1-3 complete.

**Comment by akilisosa** [comment]
tasks 4, 5, 7 complete.

**Comment by akilisosa** [comment]
If you want to swing by to give feedback in the hall about design that'd be great.

**Comment by akilisosa** [comment]
waiting on design specs, suggestions, requests from Andrew. I will probably be in the hall.

**Comment by akilisosa** [comment]
it is being merged into the dev branch with the edit categories. akili branch is clear to delete.

**Comment by akilisosa** [comment]
updates
Research the rxjs library and implementing observables and sequences into the service class or component properly. probably just adding some micro-text that says click do deactivate.
